samples/servicecontrol/adapter-rabbitmq-different-topologies/sample.md

```diff
@@ -33,12 +33,12 @@ NOTE: In order to connect to a different RabbitMQ broker, ensure all connection
 
 ## Running the project
 
- 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure adapter starts first because on start-up it creates a queue that is used for heartbeats.
+ 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure the adapter starts first because on start-up it creates a queue that is used for heartbeats.
```

Comment:
and it's not required, it comes from other samples IIRC that say the same thing. Heartbeats are sent from endpoints to SC Q, or the adapter Q, if that Q doesn't exist a warning is raised at the sender endpoint, it doesn't fail. As soon as the adapter starts Qs are created and the warning goes away. I'd entirely remove that text, replacing it with "press F5".